net.x_y.ykhwong.dosbox-daum.yml

```diff
@@ -0,0 +1,85 @@
+app-id: net.x_y.ykhwong.dosbox-daum
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: dosbox
+finish-args:         # flatpak permissions
+  - --device=all     # needed for OpenGL, gamepads and serial/parallel
+  - --share=ipc      # needed for X11
+  - --share=network  # Needed for IPX and serial over TCP/IP
+  - --socket=x11     # default to X11, as Gnome does not provide SSD
+  - --socket=pulseaudio
+  - --filesystem=home
+cleanup:
+  - '/include'
+  - '/lib/pkgconfig'
+  - '/share/aclocal'
+  - '/share/man'
+  - '*.la'
+  - '*.a'
+  - '/share/doc'
+
+modules:
+  - "shared-modules/glu/glu-9.json"
+  - "shared-modules/glew/glew.json"  
+  - name: SDL-1.2.15-hq
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/ebenbruyns/sdl1.2.15-hq.git
+        tag: release-1.2.15-hq
+  - "shared-modules/SDL/SDL_net-1.2.8.json"
+  - name: libtbb
+    buildsystem: simple
+    build-commands:
+      - make -e CXXFLAGS="-std=c++14" # -stdlib=libstdc++" 
+      - install -m755 -t /app/lib/ build/linux_*_release/*
+      - install -m755 -t /app/lib/ build/linux_*_debug/concurrent_hash_map.o # the release build of this file crashes on stripping - seems to work
+      - install -D -t /app/include/tbb/ include/tbb/*.h
+      - install -D -t /app/include/serial/tbb include/serial/tbb/*
+      - install -D -t /app/include/tbb/compat include/tbb//compat/*
+      - install -D -t /app/include/tbb/machine include/tbb/machine/*
+      - install -D -t /app/include/tbb/internal include/tbb/internal/*
+    sources:
+      - type: git
+        url: https://github.com/oneapi-src/oneTBB.git
+        tag: 2017_U8
+  - name: open-glide
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/voyageur/openglide.git
+        commit: 1ecc60a83d2255bb9ce8a3e53a5c6bcc7dce44b1   
+  - name: dosbox-daum
+    buildsystem: simple
+    build-commands:
+      - sh autogen.sh
+      - cp gl.h /app/include/GL/  
+      - ./configure --enable-core-inline --prefix=/app
+      - make -e CXXFLAGS="-std=c++11 -fpermissive -I/app/include/SDL -ltbb"
+      - make install
+    sources:
+      - type: git
+        url: https://github.com/ykhwong/dosbox-svn-daum.git
+        commit: 690080fd63ec3877d110240ccc7f0890e272c984
+          #branch: master
+      - type: file
+        path: ./gl.h
```

Comment:
this is IMHO still an issue and a bad one.

Reply:
Are you going to reject it based on that? because there's nothing I can do to fix that without making significant changes, SDL itself it patched too. Will this affect the rest of the flatpak eco system? I was under the impression that stuff like this will not affect the rest of the eco system which is specifically why I thought this is an ideal candidate for a flatpak? In an ideal world where this thing would have a long life span I'd agree with you here, but since I'm not the author of the code and this is still required to play a bunch of old dos games until dosbox staging gets it's issues resolved.

If I remove the file after building will that satisfy you? I think it's just required to build it? If you're going to reject it based on this then I'm going to reneg on this submission, it's not worth my time beyond this. I'm happy to investigate the 23.08 runtime further, but I can't do anything about this issue.

Reply:
ok this is something I'm going to need clarification on, are you going to reject it because of this, because I've burned so much time on this already, if you're going to force me to tidy up this mess and rewrite a ton of stuff then I'm going to stop here. I agree that this is bad, but I disagree that it's harmful to the entire eco system, unless you can show me why it's bad for the rest of the eco system. So I need you to either say yes or no and this submission will live or die based on that decision. I'm willing to work on the other thigns but this is not something I feel is justified to put effort in, I'm not getting paid for this nor am I getting anything else out of this at this point other than pain and punishment.

Reply:
it is acceptable now?